src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -143,6 +145,20 @@ export const buildATIPageTrackPath = ({
       value: ldpThingIds,
       wrap: true,
     },
+    {
+      key: 'x16',
+      description: 'campaigns',
+      value: (Array.isArray(campaigns) ? campaigns : [])
+        .map(campaign => campaign.campaignName.replace(/ /g, '%20'))
+        .join('~'),
```

Comment:
Do we have test cases that capture `campaign` being an array with the values joined by `~`?

Reply:
Yep - https://github.com/bbc/simorgh/pull/4222/files#diff-92bd8e49974d9252cb4eff53ee71b19aR53